packages/hs-react-ui/src/enums/colors.ts

```diff
@@ -7,6 +7,8 @@ enum colors {
   grayLight = '#7c8186',
   grayXlight = '#d3d6d7',
 
+  groupLabel = '#e8e8e8',
```

Comment:
This seems really specific - it should either named after a color descriptor or a more general use-case

Otherwise just pick a different gray or alter one from the constant file with polished (once we get custom color palletes, the table will still look right despite having a "custom" color)

Reply:
In table I moved to grayXlight :)